### ISSUE TITLE
Outdated reference to a dir in "Getting Started"

### ISSUE THREAD

**Issue by p-himik** [juxt/yada#254]
The page at https://github.com/juxt/yada/blob/master/doc/getting-started.adoc still mentions `edge/app` which now is `edge/main`, I think.
Maybe there's something else outdated in there (like running `..bin/rebel` instead of `clojure`?), I don't know.

**Comment by malcolmsparks** [comment]
Yes, it is out of date. Edge has been moving fast lately and beginning to stabilize. I fully intend to rewrite this section and link to the various yada demos that are being added to Edge.
